src/components/skills-quiz/styles/SkillsQuizDropdowns.scss

```diff
@@ -21,3 +21,9 @@
   object-fit: cover;
```

Comment:
Curious to know if we are keeping the card height fixed for select and search job cards? What would happen if one of the job titles is too long, would the height of all cards be same in that case?

Reply:
Yes, It will keep the height same even if the title is long.